main.py

```diff
@@ -302,21 +305,33 @@ def update_settings(settings: dict) -> str:
     if 'root_dir' in settings:
         root_dir = settings['root_dir']
     else:
-        root_dir = 'D:\GDrive\School 19_20'
+        # If the root_dir is not specified in the settings
+        # we default to the root path of the os. (/ for linux & c:/ or d:/ etc for windows)
+        root_dir = os.path.abspath(os.sep)

```

Comment:
Using the root of the filesystem might make the indexing really heavy and large.
It's okay for now to default to, but in the feature we might need a setup screen or a command line argmument.